src/platform/windows/display_base.cpp

```diff
@@ -1051,6 +1053,16 @@ namespace platf {
       }
     }
 
+    if (config::video.capture == "amd") {
```

Comment:
```suggestion
    if (config::video.capture == "amd" || config::video.capture.empty()) {
```

This should allot it to attempt `amd` capture if the user hasn't explicitly set anything.

What should be our automatic order? The way it is now will try in this order:

- ddx
- amd
- wgc

Reply:
I moved amd first as AMD users would prefer it over ddx because it supports AFMF.

Only downside right now is the mouse cursor not being captured.

Reply:
I think it will be fine to be first once the issues like this are ironed out. No mouse capture is a pretty big downside at the moment.